event/event.yaml

```diff
@@ -3,71 +3,55 @@ components:
     Event:
       description: |-
         The optional container for event configuration.
+        To get cp-dp data both cp_events and dp_events must be explicitly enabled. 
       type: object
       properties:
-        enable:
+        cp_events:
           description: |-
-            True to enable all events. 
-            Enabling this option may affect the resultant packet payload due to 
-            additional instrumentation data.
+            The optional container for control plane(cp) event configuration.
```

Comment:
Lets remove optional for now since enabling feature requires this container to be included.